automation_tools/__init__.py

```diff
@@ -2059,7 +2063,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     # When using VLAN Bridges os.environ.get('BRIDGE') is 'true' and
     # executes with 'interface=eth0' for Satellite6-automation.
     # When using Satellite6-installer one can specify custom interface.
-    if os.environ.get('BRIDGE') or os.environ.get('INTERFACE'):
+    if (os.environ.get('BRIDGE') or
+            os.environ.get('NAT') or os.environ.get('INTERFACE')):
```

Comment:
don't do fake libvirt sink (setup_default_capsule) also when we have NAT defined,  i.e. reasonable setup similarly to VLANs